code/modules/recycling/disposal.dm

```diff
@@ -596,7 +596,9 @@
 	for(var/obj/O in D)
 		if(O.contents)
 			for(var/mob/living/M in O.contents)
-				if(M && M.stat != 2)
+				if(istype(M, /mob/living/simple_animal/hostile/mimic/crate))
+					hasmob = 0
```

Comment:
you can't implement it like this, if you have it this way and it contains another mob besides a mimic but it checks that one first, this will reset it to 0; instead you want to `continue`